### ISSUE TITLE
Mamba 1.4.7 installed within conda base env throws libmamba ZSTD decompression error: Unknown frame descriptor

### ISSUE THREAD

**Issue by KnightArthurRen** [mamba-org/mamba#2667]
### Troubleshooting docs

- [X] My problem is not solved in the Troubleshooting docs

### How did you install Mamba?

Mambaforge

### Search tried in issue tracker

libmamba ZSTD decompression error

### Latest version of Mamba

- [X] My problem is not solved with the latest version

### Tried in Conda?

I do not have this problem with Conda, just with Mamba

### Describe your issue

We are currently building our environment by installing mamba in conda via 

`conda install -n base --override-channels -c conda-forge mamba 'python_abi=*=*cp*'`

After recent 1.4.7 update mamba installed this way broke with error msg
`error    libmamba ZSTD decompression error: Unknown frame descriptor` on our build agents

I was able to reproduce it on our linux machines by

1. Install Miniconda via https://docs.conda.io/en/latest/miniconda.html#linux-installers
2. Install mamba via `conda install -n base --override-channels -c conda-forge mamba 'python_abi=*=*cp*'`
3.`conda activate` (activated base) 
4. `mamba env create -f environment.yml` (The environment.yml below)

I tried it with mamba 1.4.5 and it looks good, I think this is related with mamba adding support to ZSTD with 1.4.7 update.
I'm wondering what should be the fix here, or should I just switch to install mamba using mambaforge?

Thank you!

### mamba info / micromamba info

```shell
mamba version : 1.4.7
     active environment : base
    active env location : /home/ubuntu/miniconda3
            shell level : 1
       user config file : /home/ubuntu/.condarc
 populated config files : /home/ubuntu/.condarc
          conda version : 23.5.0
    conda-build version : not installed
         python version : 3.11.4.final.0
       virtual packages : __archspec=1=x86_64
                          __glibc=2.31=0
                          __linux=5.11.15=0
                          __unix=0=0
       base environment : /home/ubuntu/miniconda3  (writable)
      conda av data dir : /home/ubuntu/miniconda3/etc/conda
  conda av metadata url : None
           channel URLs : https://conda.anaconda.org/conda-forge/linux-64
                          https://conda.anaconda.org/conda-forge/noarch
                          https://repo.anaconda.com/pkgs/main/linux-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/linux-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /home/ubuntu/miniconda3/pkgs
                          /home/ubuntu/.conda/pkgs
       envs directories : /home/ubuntu/miniconda3/envs
                          /home/ubuntu/.conda/envs
               platform : linux-64
             user-agent : conda/23.5.0 requests/2.29.0 CPython/3.11.4 Linux/5.11.15-xanmod1 ubuntu/20.04.4 glibc/2.31
                UID:GID : 1000:1000
             netrc file : None
           offline mode : False
```

### Logs

```shell
info     libmamba Searching index cache file for repo 'https://conda.anaconda.org/conda-forge/linux-64/repodata.json'
warning  libmamba Could not parse mod/etag header
info     libmamba Invalid json cache found, ignoring
info     libmamba No valid cache found
info     libmamba All targets to download are cached
info     libmamba Searching index cache file for repo 'https://conda.anaconda.org/conda-forge/noarch/repodata.json'
warning  libmamba Could not parse mod/etag header
info     libmamba Invalid json cache found, ignoring
info     libmamba No valid cache found
info     libmamba All targets to download are cached
info     libmamba Searching index cache file for repo 'https://conda.anaconda.org/anaconda/linux-64/repodata.json'
info     libmamba No valid cache found
info     libmamba All targets to download are cached
info     libmamba Searching index cache file for repo 'https://conda.anaconda.org/anaconda/noarch/repodata.json'
info     libmamba No valid cache found
info     libmamba All targets to download are cached
info     libmamba Searching index cache file for repo 'https://conda.anaconda.org/gurobi/linux-64/repodata.json'
info     libmamba No valid cache found
info     libmamba Starting to download targets
[+] 0.4s
gurobi/linux-64 (check zst) ━━━━━━━━━━━╸━━━━━━━━━━━━━━━╸━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━   0.0 B /  ??.?MB @  ??.?MB/s Checking  0.4sinfo     libmamba Transfer done for 'gurobi/linux-64 (check zst)'
info     libmamba Transfer finalized, status: 200 [https://conda.anaconda.org/gurobi/linux-64/repodata.json.zst] 0 bytes
gurobi/linux-64 (check zst)                         Checked  0.5s
info     libmamba Searching index cache file for repo 'https://conda.anaconda.org/gurobi/noarch/repodata.json'
info     libmamba No valid cache found
info     libmamba Starting to download targets
[+] 0.1s
gurobi/noarch (check zst) ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╸━━━━━━━━━━━━━━━╸━   0.0 B /  ??.?MB @  ??.?MB/s Checking  0.1sinfo     libmamba Transfer done for 'gurobi/noarch (check zst)'
info     libmamba Transfer finalized, status: 200 [https://conda.anaconda.org/gurobi/noarch/repodata.json.zst] 0 bytes
gurobi/noarch (check zst)                           Checked  0.2s
info     libmamba Searching index cache file for repo 'https://repo.anaconda.com/pkgs/main/linux-64/repodata.json'
info     libmamba No valid cache found
info     libmamba Expired cache (or invalid mod/etag headers) found at '/home/ubuntu/miniconda3/pkgs'
info     libmamba Starting to download targets
[+] 0.0s
info     libmamba Transfer done for 'pkgs/main/linux-64 (check zst)'
info     libmamba Transfer finalized, status: 200 [https://repo.anaconda.com/pkgs/main/linux-64/repodata.json.zst] 0 bytes
pkgs/main/linux-64 (check zst)                     Checked  0.0s
info     libmamba Searching index cache file for repo 'https://repo.anaconda.com/pkgs/main/noarch/repodata.json'
info     libmamba No valid cache found
info     libmamba Expired cache (or invalid mod/etag headers) found at '/home/ubuntu/miniconda3/pkgs'
info     libmamba Starting to download targets
[+] 0.0s
pkgs/main/noarch (check zst) ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╸━━━━━━━━━━━━━━━╸━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━   0.0 B /  ??.?MB @  ??.?MB/s Checking  0.0sinfo     libmamba Transfer done for 'pkgs/main/noarch (check zst)'
info     libmamba Transfer finalized, status: 200 [https://repo.anaconda.com/pkgs/main/noarch/repodata.json.zst] 0 bytes
pkgs/main/noarch (check zst)                        Checked  0.0s
info     libmamba Searching index cache file for repo 'https://repo.anaconda.com/pkgs/r/linux-64/repodata.json'
info     libmamba No valid cache found
info     libmamba Expired cache (or invalid mod/etag headers) found at '/home/ubuntu/miniconda3/pkgs'
info     libmamba Starting to download targets
[+] 0.0s
info     libmamba Transfer done for 'pkgs/r/linux-64 (check zst)'
info     libmamba Transfer finalized, status: 200 [https://repo.anaconda.com/pkgs/r/linux-64/repodata.json.zst] 0 bytes
pkgs/r/linux-64 (check zst)                        Checked  0.0s
info     libmamba Searching index cache file for repo 'https://repo.anaconda.com/pkgs/r/noarch/repodata.json'
info     libmamba No valid cache found
info     libmamba Expired cache (or invalid mod/etag headers) found at '/home/ubuntu/miniconda3/pkgs'
info     libmamba Starting to download targets
[+] 0.0s
pkgs/r/noarch (check zst) ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╸━━━━━━━━━━━━━━━╸━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━   0.0 B /  ??.?MB @  ??.?MB/s Checking  0.0sinfo     libmamba Transfer done for 'pkgs/r/noarch (check zst)'
info     libmamba Transfer finalized, status: 404 [https://repo.anaconda.com/pkgs/r/noarch/repodata.json.zst] 0 bytes
pkgs/r/noarch (check zst)                           Checked  0.1s
info     libmamba Starting to download targets
[+] 0.1s
conda-forge/linux-64 ━━━━╸━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━   1.3MB /  29.1MB @  22.4MB/s  0.1s
conda-forge/noarch   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╸━━━━━━━━━━━━━━━╸━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━   0.0 B /  ??.?MB @  ??.?MB/s  0.1s
anaconda/linux-64    ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╸━━━━━━━━━━━━━━━╸━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━   0.0 B /  ??.?MB @  ??.?MB/s  0.1s
anaconda/noarch      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╸━━━━━━━━━━━━━━━╸━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━   0.0 B /  ??.?MB @  ??.?MB/s  0.1s
gurobi/linux-64      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╸━━━━━━━━━━   0.0 B /  ??.?MB @  ??.?MB/s  0.1serror    libmamba ZSTD decompression error: Unknown frame descriptor
error    libmamba ZSTD decompression error: Unknown frame descriptor
info     libmamba Download error (23) Failed writing received data to disk/application [https://conda.anaconda.org/anaconda/linux-64/repodata.json.zst]
    Failure writing output to destination
anaconda/linux-64                                   ??.?MB @  ??.?MB/s 404 failed  0.2s

# >>>>>>>>>>>>>>>>>>>>>> ERROR REPORT <<<<<<<<<<<<<<<<<<<<<<

    Traceback (most recent call last):
      File "/home/ubuntu/miniconda3/lib/python3.11/site-packages/conda/exception_handler.py", line 16, in __call__
        return func(*args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^
      File "/home/ubuntu/miniconda3/lib/python3.11/site-packages/conda_env/cli/main.py", line 49, in do_call
        exit_code = getattr(module, func_name)(args, parser)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
      File "/home/ubuntu/miniconda3/lib/python3.11/site-packages/conda/notices/core.py", line 123, in wrapper
        return func(*args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^
      File "/home/ubuntu/miniconda3/lib/python3.11/site-packages/conda_env/cli/main_create.py", line 163, in execute
        result[installer_type] = installer.install(
                                 ^^^^^^^^^^^^^^^^^^
      File "/home/ubuntu/miniconda3/lib/python3.11/site-packages/mamba/mamba_env.py", line 55, in mamba_install
        index = load_channels(pool, channel_urls, repos, prepend=False)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
      File "/home/ubuntu/miniconda3/lib/python3.11/site-packages/mamba/utils.py", line 142, in load_channels
        index = get_index(
                ^^^^^^^^^^
      File "/home/ubuntu/miniconda3/lib/python3.11/site-packages/mamba/utils.py", line 123, in get_index
        is_downloaded = dlist.download(api.MAMBA_DOWNLOAD_FAILFAST)
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    RuntimeError: Download error (23) Failed writing received data to disk/application [https://conda.anaconda.org/anaconda/noarch/repodata.json.zst]
    Failure writing output to destination

`$ /home/ubuntu/miniconda3/bin/mamba create -f environment.yml -v`

  environment variables:
                 CIO_TEST=<not set>
  CONDA_AUTO_UPDATE_CONDA=false
        CONDA_DEFAULT_ENV=base
                CONDA_EXE=/home/ubuntu/miniconda3/bin/conda
             CONDA_PREFIX=/home/ubuntu/miniconda3
    CONDA_PROMPT_MODIFIER=(base)
         CONDA_PYTHON_EXE=/home/ubuntu/miniconda3/bin/python
               CONDA_ROOT=/home/ubuntu/miniconda3
              CONDA_SHLVL=1
           CURL_CA_BUNDLE=<not set>
               LD_PRELOAD=<not set>
                     PATH=/home/ubuntu/miniconda3/bin:/home/ubuntu/miniconda3/bin:/home/ubuntu/.
                          local/bin:/home/ubuntu/miniconda3/condabin:/usr/local/sbin:/usr/local/
                          bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bi
                          n
       REQUESTS_CA_BUNDLE=<not set>
            SSL_CERT_FILE=<not set>

     active environment : base
    active env location : /home/ubuntu/miniconda3
            shell level : 1
       user config file : /home/ubuntu/.condarc
 populated config files : /home/ubuntu/.condarc
          conda version : 23.5.0
    conda-build version : not installed
         python version : 3.11.4.final.0
       virtual packages : __archspec=1=x86_64
                          __glibc=2.31=0
                          __linux=5.11.15=0
                          __unix=0=0
       base environment : /home/ubuntu/miniconda3  (writable)
      conda av data dir : /home/ubuntu/miniconda3/etc/conda
  conda av metadata url : None
           channel URLs : https://conda.anaconda.org/conda-forge/linux-64
                          https://conda.anaconda.org/conda-forge/noarch
                          https://repo.anaconda.com/pkgs/main/linux-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/linux-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /home/ubuntu/miniconda3/pkgs
                          /home/ubuntu/.conda/pkgs
       envs directories : /home/ubuntu/miniconda3/envs
                          /home/ubuntu/.conda/envs
               platform : linux-64
             user-agent : conda/23.5.0 requests/2.29.0 CPython/3.11.4 Linux/5.11.15-xanmod1 ubuntu/20.04.4 glibc/2.31
                UID:GID : 1000:1000
             netrc file : None
           offline mode : False


An unexpected error has occurred. Conda has prepared the above report.
```

### environment.yml

```yaml
name: test_env
channels:
  - conda-forge
  - anaconda
  - gurobi
  - defaults
dependencies:
  - python==3.10.9
  - poetry==1.4.2
  - gdal==3.6.2
```

### ~/.condarc

```yaml
auto_activate_base: false
channels:
  - conda-forge
  - defaults
```

**Comment by JohanMabille** [comment]
This issue has been fixed on master. I have another important fix to push and I will release a new version (tonight or tomorrow morning).

**Comment by KnightArthurRen** [comment]
> This issue has been fixed on master (see #2661). I have another important fix to push and I will release a new version (tonight or tomorrow morning).
Awesome thank you!
